Source:   Gentoo
Upstream: No
Reason:   Fix wrong character classes no longer misinterpreted by grep 2.7
          ( See: http://bugs.gentoo.org/338853 )

--- docbook-utils-0.6.14/bin/jw.in
+++ docbook-utils-0.6.14/bin/jw.in
@@ -80,9 +80,9 @@
 SGML_CATALOGS_DIR="/etc/sgml"
 if [ -f "$SGML_CONF" ]
 then
-  RE='^[:space:]*SGML_BASE_DIR[:space:]*=[:space:]*'
+  RE='^[[:space:]]*SGML_BASE_DIR[[:space:]]*=[[:space:]]*'
   SGML_BASE_DIR=`grep $RE $SGML_CONF | sed "s/$RE//"`
-  RE='^[:space:]*SGML_CATALOGS_DIR[:space:]*=[:space:]*'
+  RE='^[[:space:]]*SGML_CATALOGS_DIR[[:space:]]*=[[:space:]]*'
   SGML_CATALOGS_DIR=`grep $RE $SGML_CONF | sed "s/$RE//"`
 fi
 
